cli/src/test/resources/build.gradle

```diff
@@ -36,10 +36,10 @@ task api(type: Javadoc) {
     }
 }
 
-System.out.println("Environment: " + System.getenv())
 if ( System.getenv("JAVA_HOME") == null ) {
     throw new RuntimeException()
 }
+System.out.println("Environment: JAVA_HOME=" + System.getenv("JAVA_HOME"))
```

Comment:
Not sure what this change is related to, but I see you're checking for it in the logs during the tests?

Reply:
Your change to pass the full environment through broke this test on my machine which has a coloured prompt - I resolved it by limiting its printout to just JAVA_HOME instead of the entire env.

Reply:
I think you're referring to `DifferentJvmTest:224` failing?

We could possibly use logging at `Main.java:205-211` where it does the actual `JAVA_HOME` check.

Reply:
The DifferentJVMTest was failing due to colours being printed by the `getenv` call. Simplifying this test resolves it.